packages/expo-device/src/ExpoDevice.web.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  get name(): string {
+    return "ExpoDevice";
+  }
+}
```

Comment:
I think atm the convention is to add new lines at the end of files — please ensure you've prettified the files